spec/index.html

```diff
@@ -2360,6 +2077,12 @@ <h2>Substantive changes since RDF 1.1</h2>
       It is recommended that <code>rdf:PlainLiteral</code> not be used in RDF.
     </li>
 
+    <li> <span id="whatnot"></span> <span id="non_semantics"></span> <span id="Reif"></span> <span id="containers"></span> <span id="collections"></span> <span id="rdf-collections"></span>
+      The appendix on RDF reification, containers and collections has been removed because it had no semantic content.
+      The vocabulary involved is still described in
+      <a href="https://www.w3.org/TR/rdf12-schema/#ch_othervocab">the Other Vocabulary section of RDF Schema</a>
```

Comment:
```suggestion
      The appendix on RDF reification, containers, and collections has been removed, because it had no semantic content.
      The vocabulary involved is still described in
      <a href="https://www.w3.org/TR/rdf12-schema/#ch_othervocab">the Other Vocabulary section of RDF Schema</a>.
```